opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -282,6 +284,99 @@ export const filtersWithEntityType = (
   };
 };
 
+// remove filter with key=entity_type and values contains 'all'
+// because in this case we want everything, so no need for filters
+export const removeAllTypesFromFilter = (inputFilters?: FilterGroup) => {
+  if (inputFilters && isFilterGroupNotEmpty(inputFilters)) {
+    const { filters, filterGroups } = inputFilters;
+    const newFilters: Filter[] = [];
+    const newFilterGroups: FilterGroup[] = [];
+    filters.forEach((filter) => {
+      if (!(filter.key === 'entity_type' && filter.values.includes('all'))) {
+        newFilters.push(filter);
+      }
+    });
+    filterGroups.forEach((group) => {
+      const newGroup = removeAllTypesFromFilter(group);
+      if (newGroup) filterGroups.push(newGroup);
+    });
```

Comment:
I think, you can use map instead and a filter to remove all `undefined ` if needed but I don't see a case where `removeAllTypesFromFilter ` can return undefined as we checked before ?
```
filterGroups.map((group) =>  removeAllTypesFromFilter(group))
```

Reply:
fixed